azurerm/internal/services/compute/virtual_machine_resource.go

```diff
@@ -680,6 +687,9 @@ func resourceArmVirtualMachineCreateUpdate(d *schema.ResourceData, meta interfac
 	networkProfile := expandAzureRmVirtualMachineNetworkProfile(d)
 	vmSize := d.Get("vm_size").(string)
 	properties := compute.VirtualMachineProperties{
+		SecurityProfile: &compute.SecurityProfile{
+			EncryptionAtHost: utils.Bool(d.Get("encryption_at_host").(bool)),
+		},
```

Comment:
The `azurerm_virtual_machine` resource is now in feature-frozen state, therefore could we revert the feature changes in this file?